swift/Sources/Blockchains/Cosmos.swift

```diff
@@ -0,0 +1,32 @@
+// Copyright © 2017-2019 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+import Foundation
+
+public class Cosmos: Blockchain {
+
+    public override init(purpose: Purpose = .bip44) {
+        super.init(purpose: purpose)
+    }
+
+    public let hrp: HRP = .cosmos
+
+    override public var coinType: CoinType {
```

Comment:
can you reuse `Binance Chain` class? it's the same class

Reply:
We have removed `Blockchain` and `BlockchainFactory`, could you please rebase

Reply:
@vikmeup @hewigovens @alejandro-isaza 
iOS side of things are cleaned up, and testSigningTransaction() is passing.
Please advise what's next, thanks.